### PR TITLE
Add a simple Pull Request template

### PR DESCRIPTION
**Pull request by Huang-Wei**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. If this is your first time, read our contributor guidelines https://git.k8s.io/community/contributors/guide/pull-requests.md#the-pull-request-submit-process and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
+2. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
+3. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/pull-requests.md#write-release-notes-if-needed
+4. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+-->
+
```
Comment (denkensk) [inline .github/PULL_REQUEST_TEMPLATE.md:7]:
```suggestion
#### What type of PR is this?

<!--
Add one of the following kinds:
/kind bug
/kind cleanup
/kind documentation
/kind feature

Optionally add one or more of the following kinds if applicable:
/kind api-change
/kind deprecation
/kind failing-test
/kind flake
/kind regression
-->
```

**Review by denkensk — COMMENTED** [review]
Added an essential question for the type of pr.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Huang-Wei) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: I was thinking of a simplified version... But anyway, copied the version from k8s.io/kubernetes.

**Comment by denkensk** [comment]
/lgtm

Thanks. @Huang-Wei

**Pull request merged**